addons/account_edi_ubl_cii_tax_extension/models/account_tax.py

```diff
@@ -85,13 +85,15 @@ class AccountTax(models.Model):
             ('VATEX_FR-CNWVAT', 'VATEX-FR-CNWVAT - France domestic Credit Notes without VAT, due to supplier forfeit of VAT for discount'),
         ]
     )
+    ubl_cii_requires_exemption_reason = fields.Boolean(compute="_compute_ubl_cii_requires_exemption_reason")
```

Comment:
(nitpicking but won't block for that): Habib and Laurent told me a while ago that attribute that the user won't see are generally in single quotes :smile: